File: Baekjoon_Online_Judge/04_array/03_boj_2577.py
num = [0] * 10
a = int(input())
b = int(input())
c = int(input())
n = int(a*b*c)

while n != 0:
	num[n%10] += 1
	n = int(n/10)

for item in num:
	print(item)

File: Baekjoon_Online_Judge/04_array/03_boj_2577.cpp
#include <iostream>

int main(){
    int num[10] = {0,};
    int a, b, c;
    scanf("%d\n%d\n%d", &a, &b, &c);
    long n = a*b*c;
    while(true){
        if(n == 0)
            break;
        num[n%10] += 1;
        n /= 10;
    }
    for(int i = 0; i<10; i++)
        printf("%d\n", num[i]);
}